### PR TITLE
add pvpbeach

### PR DESCRIPTION
**Pull request by ohemilyy**
### General:
* [x] The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated servers.json`)*
* [x] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:
* [ ] No changes were made to the file's formatting.
* [ ] There are no syntax errors.
* [ ] There is no pre-existing mapping matching my id.
* [ ] My field values match your requirements.
* You can view our patterns here: [servers.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/servers.schema.json), or take a look below and complete the field checklist:
  - [ ] `id`: a lowercase string *(ex. `myserver`)*
  - [x] `name`: a string *(ex. `MyServerPvP`)*
  - [x] `addresses`: an array with lowercase strings *(ex. `["my.server", "your-server.com"]`)*
    - You do not need to specify subdomains, Lunar Client services automatically detect them.
  - [x] `primaryAddress`: an address *(ex. `my.server`)*
  - [ ] `minecraftVersions`: an array with Minecraft versions as strings *(ex. `["1.18", "1.19"]`)*
  - [ ] `primaryMinecraftVersion`: a Minecraft version as a string *(ex. `1.18`)*

### Logo Additions or Updates:
* [ ] My image is a `png` file.
* [ ] I uploaded my image to the `logos` directory.
* [ ] My image has a transparent background and is square (1:1 aspect ratio).
* [ ] My image is `512` pixels in width and height.
* [ ] My image's file name matches its mapping id.

### CODE REVIEW

File: servers.json [hunk 1]
```diff
@@ -37,6 +37,18 @@
             "glade.gg"
         ]
     },
+    },
```
Comment (itsJordan) [inline servers.json:40]:
```suggestion
```

**Review by itsJordan — CHANGES_REQUESTED** [review]
Please apply my suggestion on line 40, include all supported Minecraft Versions inside the minecraftVersions array, and adjust your logo to be 512x512 pixels.

File: servers.json [hunk 2]
```diff
@@ -37,6 +37,18 @@
...
+    {
+        "id": "PvPBeach",
```
Comment (itsJordan) [inline servers.json:42]:
```suggestion
        "id": "pvpbeach",
```

**Comment by ohemilyy** [comment]
> Please apply my suggestion on line 40, include all supported Minecraft Versions inside the minecraftVersions array, and adjust your logo to be 512x512 pixels.

done

File: servers.json [hunk 3]
```diff
@@ -37,6 +37,17 @@
...
+        "id": "pvpbeach",
+        "name": "PvPBeach Network",
+        "addresses": [
+            "pvpbeach.com"
+        ],
+        "primaryAddress": "pvpbeach.com",
+        "minecraftVersions": ["1.7", "1.8", "1.9", "1.10", "1.11", "1.12"],
+        "primaryMinecraftVersion": "1.8"
+    }
+]
```
Comment (itsJordan) [inline servers.json:50]:
```suggestion
```

File: servers.json [hunk 4]
```diff
@@ -37,6 +37,17 @@
...
+    {
+        "id": "pvpbeach",
+        "name": "PvPBeach Network",
+        "addresses": [
+            "pvpbeach.com"
+        ],
+        "primaryAddress": "pvpbeach.com",
+        "minecraftVersions": ["1.7", "1.8", "1.9", "1.10", "1.11", "1.12"],
+        "primaryMinecraftVersion": "1.8"
+    }
```
Comment (itsJordan) [inline servers.json:49]:
```suggestion
    },
```

**Review by itsJordan — CHANGES_REQUESTED** [review]
Your json is invalid.

Please review my suggested changes.

**Comment by ohemilyy** [comment]
Now?

**Comment by itsJordan** [comment]
> Now?

Line 50 still contains an invalid bracket.

**Comment by ohemilyy** [comment]
Should be fixed

File: servers.json [hunk 5]
```diff
@@ -37,6 +37,16 @@
...
         ]
     },
+    {
+        "id": "pvpbeach",
+        "name": "PvPBeach Network",
+        "addresses": [
+            "pvpbeach.com"
+        ],
+        "primaryAddress": "pvpbeach.com",
+        "minecraftVersions": ["1.7", "1.8", "1.9", "1.10", "1.11", "1.12"],
```
Comment (itsJordan) [inline servers.json]: Only the following versions are supported by Lunar:
"1.7", "1.8", "1.12", "1.16", "1.17", "1.18", "1.19"

Please remove 1.9, 1.10, and 1.11.

**Review by itsJordan — APPROVED** [review]

**Pull request merged**